articles/white-hat-seo-black-hat-seo/index.md

```diff
@@ -0,0 +1,71 @@
+
+- [Semrush](https://www.semrush.com/)
+It is another one of the most popular SEO tools that many people use. It helps with SEO management, marketing, advertising, campaigning, etc. It also provides PPC (Pay-per-click) and SMM (Social Media Marketing) services, along with keyword research, link building, website survey, and many more. Users can choose from a variety of toolkits to meet their specific requirements. The SEO toolkit, for example, addresses site evaluation, thorough site research, On-Page, and Off-Page SEO Checker, organic results research, and more. The Advertising toolkit helps in content advertising, domain overview, services which include PLA(Product Listing Ads) and CPC(Cost-per-click), etc. Semrush may be a useful tool for website optimization and general business development as well. Learn more about it [here](https://www.semrush.com/kb/).
+
+- [Google Analytics](https://analytics.google.com/analytics/web/)
+Google Analytics is a web analytics tool offered by Google that provides basic data and analysis regarding website traffic. Google Analytics is used to measure website activities such as session duration, pages per session, bounce rate, etc. It also supports real-time analytics, allowing a user to gain information about visitors who are actively on the site. One of the best Google Analytics features is that it can identify badly performing sites based on information such as where visitors came from, how long they remained on the website, and their geographical location. This helps the user to improve the website qualitatively based on the reports. More details about Google Analytics can be found [here](https://en.wikipedia.org/wiki/Google_Analytics).
+
+- [Google Search Console](https://search.google.com/search-console/welcome)
+Formerly known as &quot;Google webmaster tools&quot;, google search console is a free Google tool that allows you to build, maintain, and troubleshoot your website&#39;s visibility in Google Search results. It offers information on how Google crawls, indexes, and delivers web pages. This can assist website owners in monitoring and optimizing search performance. It helps you check the security issues with the website. This helps in avoiding malware attacks and site hacking. It also allows you to specify a preferred domain for your site, which determines how the site URL is displayed in SERPs. It helps to see what keyword searches on Google lead to the site being featured in the SERPs. Learn more about Google Search Console [here](https://en.wikipedia.org/wiki/Google_Search_Console) and [here](https://www.section.io/engineering-education/google-search-console-introduction/) is a simple tutorial on how to use the same.

```

Comment:
```suggestion
Formerly known as &quot;Google webmaster tools&quot;, Google search console is a free Google tool that allows you to build, maintain, and troubleshoot your website&#39;s visibility in Google Search results. It offers information on how Google crawls, indexes, and delivers web pages. This tool assists website owners in monitoring and optimizing search performance. It helps you check the security issues with the website. This helps to avoid malware attacks and site hacking. It also allows you to specify a preferred domain for your site, which determines how the site URL is displayed in SERPs. It helps to see what keyword searches on Google lead to the site being featured in the SERPs. Learn more about Google Search Console [here](https://en.wikipedia.org/wiki/Google_Search_Console) and [here](https://www.section.io/engineering-education/google-search-console-introduction/) is a simple tutorial on how to use the same.
```